.eslintrc.json

```diff
@@ -71,7 +71,8 @@
           }
         ],
         "@typescript-eslint/explicit-function-return-type": "off",
-        "@typescript-eslint/no-unused-vars": "off"
+        "@typescript-eslint/no-unused-vars": "off",
+        "@typescript-eslint/no-non-null-assertion": "off"
```

Comment:
```suggestion
```

This rule is intentionally enabled on this repo.

Actually, when did `no-unused-vars` get disabled>?

Reply:
You can disable specific cases if need with comments, or just add a `if (x == null) throw new Error(..)` statement